tests/matrix_free/matrix_vector_faces_26.output.haswell

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Please rename into `.avx` which is consistent with other output files, e.g. `tests/matrix_free/cell_categorization.with_p4est=true.mpirun=1.output.avx`.